uploadNeuroDB/NeuroDB/ImagingUpload.pm

```diff
@@ -410,19 +411,44 @@ Arguments:
 sub PatientNameMatch {
     my $this         = shift;
     my ($dicom_file) = @_;
-    my $cmd          = "dcmdump $dicom_file | grep PatientName";
+
+    my $lookupCenterNameUsing = NeuroDB::DBI::getConfigSetting(
+        $this->{'dbhr'},'lookupCenterNameUsing'
+    );
+
+    unless ( $lookupCenterNameUsing ) {
+        my $message = "\nConfig Setting 'lookupCenterNameUsing' is not set in "
+                      . "the Config module under the Imaging Pipeline section.";
+        $this->spool($message, 'Y', $notify_notsummary);
+        #TODO: once refactoring of exit code merged to minor, add the following
+        #TODO: MISSING_CONFIG_SETTING exit code in the generic database code
+        exit $NeuroDB::ExitCodes::MISSING_CONFIG_SETTING;
```

Comment:
This still has to be defined in ExitCode.pm

Reply:
good point @nicolasbrossard 
I did not realize this is what you have been telling me since yesterday!

otherwise, I tested it (by manually adding that missing exit code myself based on your comment), and it works...
so as soon as @cmadjar updates her PR, I will add the passed manual test tag